engine/src/main/java/org/terasology/network/internal/ClientHandler.java

```diff
@@ -66,6 +77,10 @@ public void exceptionCaught(ChannelHandlerContext ctx, ExceptionEvent e) {
     }
 
 
+    /**
+     *  Marks the joining of the server to be complete and sets the server in networkSystem.
```

Comment:
you have an extra space here before you're message.

Reply:
I go back through files as I write these and update them all again once I've physically seen them being called. Most of these are placeholders while I keep working so I know where to go back and fix later.